apiserver/Makefile

```
@@ -232,7 +227,7 @@ run-kubernetes-server: config/crd run-etcd stop-kubernetes-server
 		-v $(CURDIR)/test/certs:/home/user/certs \
```

I can't add a suggestion directly but it seems like you might need to modify the manifests volume above this line to: 
```suggestion
                -v $(CURDIR)/../:/go/src/github.com/projectcalico/calico:rw
		-v $(CURDIR)/test/certs:/home/user/certs \
```